modules/installation-vsphere-machines.adoc

```diff
@@ -16,24 +16,49 @@ Before you install a cluster that contains user-provisioned infrastructure on VM
 
 .Procedure
 
-. Convert the control plane, compute, and bootstrap Ignition config files to Base64 encoding.
-
+. Locate the following Ignition config files that the installation program created, for example:
++
+* `<installation_directory>/bootstrap.ign`
+* `<installation_directory>/master.ign`
+* `<installation_directory>/worker.ign`
++
+. Convert the bootstrap Ignition config file to Base64 encoding and store it in the extra configuration properties of the virtual machine (VM) before booting.
 +
-For example, if you use a Linux operating system, you can use the `base64` command to encode the files.
+* To convert to Base64 and change the bootstrap Ignition config file, enter the following commands:
++
+[source,terminal]
+----
+$ govc vm.change -vm "<vm_name>" -e "guestinfo.ignition.config.data.encoding=base64"
+----
++
+[source,terminal]
+----
+$ govc vm.change -vm "<vm_name>" -e "guestinfo.ignition.config.data=$(cat bootstrap.ign)"
+----
++
+. Convert the control plane, or master, Ignition config files to Base64 encoding and store them in the extra configuration properties of the control plane VMs before booting.
++
+.. To convert to Base64 and change the control plane, or master, Ignition config files, enter the following commands for each `master.ign` file:
++
+[source,terminal]
+----
+$ govc vm.change -vm "<vm_name>" -e "guestinfo.ignition.config.data.encoding=base64"
+----
 +
 [source,terminal]
 ----
-$ base64 -w0 <installation_directory>/master.ign > <installation_directory>/master.64
+$ govc vm.change -vm "<vm_name>" -e "guestinfo.ignition.config.data=$(cat master.ign)"
```

Comment:
Followup to Micah's comment https://github.com/openshift/openshift-docs/pull/28106/files#r635516210, it would look like this for the `base64` conversion

```suggestion
$ govc vm.change -vm "<vm_name>" -e "guestinfo.ignition.config.data=$(base64 -w0 master.ign)"
```

Reply:
Thanks, @mike-nguyen - you read my mind! I've resolved the 3 places where this occurs.